course/flow.py

```diff
@@ -2107,6 +2114,39 @@ def view_flow_page(
     # }}}
 
 
+@course_view
+def view_flow_tab_page(
```

Comment:
One thing I'm concerned about is that students in the CBTF (who can use native tabs) may complain about the decrease in vertical space available. I was thinking that maybe this could be enabled on request, by clicking a link.

Reply:
Agree, then in this PR i will only add the route for the tabeed page. To enter it, either student manual change the url, or provide a hyperlink in the html page (in the exam flow configuration) which redirect to the tabbed page